### ISSUE TITLE
discu.eu weekly newsletters not added

### ISSUE THREAD

**Issue by bedlamzd** [omnivore-app/omnivore#2834]
I've subscribed to multiple newsletters (https://discu.eu/weekly/), got email for each from discu.eu with confirmation link (to my personal email, by the way, I assume that shouldn't have happened).

But none of the subscriptions appeared in my list and no letters were received.

Nice project! Hope it will get going, I look forward to replacing my other services with it someday

**Comment by jacksonh** [comment]
Hi, confirmation emails will often be forwarded so you can confirm them. 

Can you login on the web and check for  these newsletters on this page: https://omnivore.app/settings/emails/recent

**Comment by bedlamzd** [comment]
yep, here they are

![image](https://github.com/omnivore-app/omnivore/assets/44726005/aa1b791c-7daa-4c0d-a6cf-8d9c6497588b)

I've subscribed to more, but I assume they didn't arrive yet

**Comment by bedlamzd** [comment]
ah, I see, I mistook these two for new issues, sorry.

**Comment by jacksonh** [comment]
Yeah so our logic is if we don't think the received email is an article we will forward it to you to allow you to do something like click a link if needed. You can also open the emails received on that view. 

If they publisher hasn't sent an issue out yet, it wont show up in your library. Once you do receive one if it works out let us know.

**Comment by bedlamzd** [comment]
The newsletter should be alive, I see new issues on their site and mastodon.

Could it be that it's blocked somehow? I've been using kill-the-newsletter before for them and stopped receiving letters a while ago.

Some newsletter blocked certain email domains as well

**Comment by jacksonh** [comment]
Perhaps, i can look closer at our logs but if there aren't issues in the recent emails section its unlikely we have received anything. I assume you did click the link to confirm?

**Comment by bedlamzd** [comment]
Yep, and, when I've tried to subscribe again, newsletter recognized my email and said I'm already confirmed.

Don't really want to waste your time though, it's quite possible that newsletter itself is broken for some reason.

I don't think I can reach out to them, because support is behind premium plan and their reddit seems dead.

For now I've solved the issue by subscribing to mastodon rss for the same feed.

I'll close this issue. Thank you for your time!
